packages/foam-vscode/src/core/services/markdown-link.test.ts

```diff
@@ -254,4 +254,140 @@ describe('MarkdownLink', () => {
       expect(edit.range).toEqual(link.range);
     });
   });
+
+  describe('convert wikilink to link', () => {
+    it('should generate default alias if no one', () => {
+      const wikilink = parser.parse(getRandomURI(), `[[wikilink]]`).links[0];
+      const wikilinkEdit = MarkdownLink.createUpdateLinkEdit(wikilink, {
+        type: 'link',
+      });
+      expect(wikilinkEdit.newText).toEqual(`[wikilink](wikilink)`);
```

Comment:
Probably by default here we should add the default extension. 

Which makes me think..
I almost wonder whether, when we change `type` we also MUST provide the `target`. The pro is that the fn is much more stable as we really cannot guess from wikilink to link and viceversa in any reliable way. The con is that the API of the fn is a bit less straightforward. 
Wdyt?

Reply:
I know what you mean is that the`createUpdateLinkEdit` is functional but not smart enough. Regardless of its name, it can change any parts of a link and re-organize it in a different format. To provide more fancy functions, I think it just needs another layer of API.